src/data.rs

```diff
@@ -250,12 +250,22 @@ impl Data {
         let (_start_x, _start_y) = (0, 0);
         let (block_start_x, mut block_start_y) = (0, 0);
         let workbook = Workbook::new(&format!("{}.xlsx", file_name));
+        let title_format = workbook.add_format()
+            .set_font_size(16.0)
+            .set_bold();
+        let head_row_format = workbook.add_format()
+            .set_text_wrap()
+            .set_align(FormatAlignment::CenterAcross)
+            .set_border(FormatBorder::Thin)
+            .set_bg_color(FormatColor::Cyan);
+        let data_row_format = workbook.add_format()
+            .set_text_wrap()
+            .set_border(FormatBorder::Thin);
```

Comment:
Set top alignment for vertical alignment (current: bottom alignment)

Reply:
Done :)